relay/api/tokens/resources.py

```diff
@@ -0,0 +1,102 @@
+from relay.blockchain.unw_eth_proxy import UnwEthProxy
+from relay.blockchain.token_proxy import TokenProxy
+
+from relay.api.schemas import TokenEventSchema, UserTokenEventSchema
+from flask_restful import Resource
+from flask import abort
+from webargs import fields
+from webargs.flaskparser import use_args
+from marshmallow import validate
+from typing import Union  # NOQA
+from relay.relay import TrustlinesRelay
+
+
+def abort_if_unknown_token(trustlines, token_address):
+    if token_address not in trustlines.token_addresses and token_address not in trustlines.unw_eth_addresses:
+        abort(404, 'Unkown network: {}'.format(token_address))
+
+
+class TokenAddresses(Resource):
+
+    def __init__(self, trustlines):
+        self.trustlines = trustlines
+
+    def get(self):
+        return self.trustlines.unw_eth_addresses + self.trustlines.token_addresses
```

Comment:
if you always treat it like this, self.trustlines.token_addresses should just include unw_eth_addresses already, or are there exceptions?

Reply:
hmm cant think of any exceptions at this moment... will include it.